lib/llvm-backend/src/backend.rs

```diff
@@ -300,9 +423,30 @@ impl RunnableModule for LLVMBackend {
             mem::transmute(symbol)
         };
 
+        SIGNAL_HANDLER_INSTALLED.call_once(|| unsafe {
+            crate::platform::install_signal_handler();
+        });
+
         Some(unsafe { Wasm::from_raw_parts(trampoline, invoke_trampoline, None) })
     }
 
+    fn get_code(&self) -> Option<&[u8]> {
+        Some(unsafe {
+            std::slice::from_raw_parts(
+                llvm_backend_get_code_ptr(self.module),
+                llvm_backend_get_code_size(self.module),
+            )
+        })
+    }
+
+    fn get_local_function_offsets(&self) -> Option<Vec<usize>> {
+        Some(self.local_func_id_to_offset.clone())
+    }
+
+    fn get_module_state_map(&self) -> Option<ModuleStateMap> {
+        self.msm.clone()
+    }
```

Comment:
I'd expect these to return references based on the name 🤔 though it's probably fine